index.js

```diff
@@ -155,7 +162,17 @@ module.exports = function(sails) {
       before: {
         '/*': function(req, res, next) {
           if (injectRequestLogger) {
-            req.log = _this.logger.child({req: req}, true);
+
+            var options = {req: req};
+
+            if (generateRequestId) {
+
+              // jscs: disable
+              options.req_id = uuid.v4(); // jshint ignore:line
```

Comment:
Yeah that's a good idea.  Will add.  Necessary to pass along to other service calls.